source/connect/favorite-connections/import-export-ui/import.txt

```diff
@@ -0,0 +1,56 @@
+.. _compass-import-connections-ui:
+
+==============================================
+Import Favorite Connections on |compass-short|
+==============================================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can import favorite connections on |compass|. This lets you use saved 
+connections from other workspaces or connections used by other team members.
+
+Prerequisite
+~~~~~~~~~~~~
+
+Before you import favorite connections, you must :ref:`export the
+connections <compass-export-connections-ui>`.
+
+Procedure
+~~~~~~~~~
+
+To import favorite connections from a JSON file into |compass-short|:
+
+.. procedure::
+    :style: normal
+    
+    .. step:: Hover over :guilabel:`Saved Connections` in the left-side 
+        navigation and click the :guilabel:`Ellipses (...)` button.
+
+    .. step:: Select :guilabel:`Import saved connections` to open the import 
+        dialog box.
+
+    .. step:: Under :guilabel:`Source File`, click :guilabel:`Select a file...` 
+        to select the exported JSON file you want to import.
```

Comment:
I feel like we can make this more active voice and direct by splitting this into two sentences:

Under :guilabel:`Source File`, click :guilabel:`Select a file...`. Select the exported JSON file you want to import.

You could make these separate steps or leave them as one, up to you.

Reply:
Noted and updated! Opting to split.